### PR TITLE
Linkfixes

### LINKED ISSUE #1841: Secondary indexing - Image links is broken
image

### LINKED ISSUE #1866: Redis pipelining
Hi
I found some pictures missing when I read this [document](https://redis.io/docs/manual/pipelining/)
Screenshot from 2022-04-11 14-35-43

### PR DESCRIPTION
**Pull request by nermiller**
Fixes #1841
Fixes #1866

### CODE REVIEW

File: clients/php/github.com/Shumkov/Rediska.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
     "name": "Rediska",
     "description": "A Redis client.",
-    "homepage": "http://rediska.geometria-lab.net",
```
Comment (banker) [inline clients/php/github.com/Shumkov/Rediska.json:4]: @nermiller If the code links is dead, can you remove this file entirely?

File: docs/manual/data-types/_index.md [hunk 2]
```diff
@@ -152,4 +152,4 @@ A Redis stream is a data structure that acts like an append-only log. Streams ar
 
 Redis provides geospatial indexes, which are useful for finding locations within a given geographic radius. You can add locations to a geospatial index using the [GEOADD](/commands/geoadd) command. You then search for locations within a given radius using the [GEORADIUS](/commands/georadius) command.
 
-See the [complete geospatial index command reference](commands/?group=geo) for all of the details.
+See the [complete geospatial index command reference](https://redis.io/commands/?group=geo) for all of the details.
```
Comment (banker) [inline docs/manual/data-types/_index.md:155]:
```suggestion
See the [complete geospatial index command reference](/commands/?group=geo) for all of the details.
```

File: docs/reference/patterns/twitter-clone.md [hunk 3]
```diff
@@ -14,15 +14,15 @@ applications using Redis as their main store, so the goal of the article today
 is to be a tutorial for Redis newcomers. You'll learn how to design a simple
 data layout using Redis, and how to apply different data structures.
 
-Our Twitter clone, called [Retwis](http://retwis.antirez.com), is structurally simple, has very good performance, and can be distributed among any number of web and Redis servers with little efforts. You can find the source code [here](http://code.google.com/p/redis/downloads/list).
+Our Twitter clone, called Retwis, is structurally simple, has very good performance, and can be distributed among any number of web and Redis servers with little efforts. You can find the source code [here](http://code.google.com/p/redis/downloads/list).
```
Comment (banker) [inline docs/reference/patterns/twitter-clone.md:17]:
```suggestion
Our Twitter clone, called [Retwis](https://github.com/antirez/retwis), is structurally simple, has very good performance, and can be distributed among any number of web and Redis servers with little efforts. [View the Retwis source code](https://github.com/antirez/retwis).
```

**Review by banker — CHANGES_REQUESTED** [review]
Looks good, but I think we need to remove rediska altogether, plus a couple of other minor issues.

File: docs/manual/eviction.md [hunk 4]
```diff
@@ -106,10 +106,7 @@ What is important about the Redis LRU algorithm is that you **are able to tune**
 
 The reason Redis does not use a true LRU implementation is because it
 costs more memory. However, the approximation is virtually equivalent for an
-application using Redis. The following is a graphical comparison of how
-the LRU approximation used by Redis compares with true LRU.
-
-![LRU comparison](https://redis.io/images/redisdoc/lru_comparison.png)
+application using Redis.
```
Comment (itamarhaber) [inline docs/manual/eviction.md:109]: The file still exists (https://github.com/redis/redis-io/blob/master/public/images/redisdoc/lru_comparison.png) - lets re-add it (and others maybe?)

File: docs/reference/patterns/indexes.md [hunk 5]
```diff
@@ -547,15 +547,12 @@ sometimes used. Here we'll describe a different way to index data into
...
-Let's start by visualizing the problem. In this picture we have points
-in the space, which represent our data samples, where `x` and `y` are
+Let's say we have points in the space, which represent our data samples, where `x` and `y` are
 our coordinates. Both variables max value is 400.
 
-The blue box in the picture represents our query. We want all the points
+We want all the points
 where `x` is between 50 and 100, and where `y` is between 100 and 300.
 
-![Points in the space](https://redis.io/images/redisdoc/2idx_0.png)
```
Comment (itamarhaber) [inline docs/reference/patterns/indexes.md:557]: Same same

File: docs/reference/patterns/indexes.md [hunk 6]
```diff
@@ -591,9 +588,7 @@ We obtain a range which is lexicographically continuous:
 What this maps to is to a square representing all values where the `x`
 variable is between 70 and 79, and the `y` variable is between 200 and 209.
 We can write random points in this interval, in order to identify this
-specific area:
-
-![Small area](https://redis.io/images/redisdoc/2idx_1.png)
+specific area.
```
Comment (itamarhaber) [inline docs/reference/patterns/indexes.md:591]: Ditto

File: clients/php/github.com/Shumkov/Rediska.json (hunk 1 above)
Comment (nermiller) [reply clients/php/github.com/Shumkov/Rediska.json:4]: will do

**Comment by nermiller** [comment]
@banker please review when you can. Thanks!

File: docs/about/license.md [hunk 7]
```diff
@@ -51,9 +51,9 @@ Redis uses source code from third parties. All this code contains a BSD or BSD-c
 
 * Redis uses the `sha1.c` file that is copyright by Steve Reid and released under the **public domain**. This file is extremely popular and used among open source and proprietary code.
 
-* When compiled on Linux Redis uses the [Jemalloc allocator](http://www.canonware.com/jemalloc/), that is copyright by Jason Evans, Mozilla Foundation and Facebook, Inc and is released under the **two-clause BSD license**.
+* When compiled on Linux, Redis uses the [Jemalloc allocator](https://github.com/jemalloc/jemalloc), which is copyrighted by Jason Evans, Mozilla Foundation, and Facebook, Inc and released under the **two-clause BSD license**.
 
-* Inside Jemalloc the file `pprof` is copyright Google Inc and released under the **three-clause BSD license**.
+* Inside Jemalloc, the file `pprof` is copyright by Google Inc and released under the **three-clause BSD license**.
```
Comment (banker) [inline docs/about/license.md:56]:
```suggestion
* Inside Jemalloc, the file `pprof` is copyrighted by Google Inc and released under the **three-clause BSD license**.
```

**Pull request merged**